README.md

```diff
@@ -1,8 +1,30 @@
-# Final-Project-Template
-<!-- Edit the title above with your project title -->
+# The Sustainability of Countries' Development Over Time
 
 ## Project Overview
 
+### Research Questions
+1) Is the sustainability of a country related to its development and GDP?
+2) How have these trends differed over time?
+
+### Motivation
+First, I would like to confirm that the development of a country and its GDP are correlated with its overall carbon emission levels.
```

Comment:
Absolutely, it's hypocritical for a developed country to criticize a developing country for doing the same thing they did a few decades earlier.
My intention isn't to cast blame on developing countries, or tell them what to do. I just want to gain a better understanding of the trends to assess risk and *hopefully* encourage more sustainable development.

This century, there is more awareness about environmental issues than in the past. I hope that this knowledge allows us to avoid repeating the mistakes of the past, but I won't prescribe any particular policy for development. It's not my job to tell a country how they should develop.